app/[locale]/job/list/page.tsx

```diff
@@ -4,17 +4,25 @@ import { Button, Grid, GridContainer } from '@trussworks/react-uswds'
 import { useTranslation } from 'react-i18next'
 import { useRouter } from "next/navigation"
 import { useAppSelector } from "@/lib/hooks"
+
+import { recommendStandardDeduction } from "@/lib/store"
+import { selectJobItems } from '@/lib/features/job/jobSlice'
 import IncomeList from "@/app/components/IncomeList"
 import VerifyNav from "@/app/components/VerifyNav"
-import { recommendStandardDeduction } from "@/lib/store"
 
 const DAY_COUNT = 30
 
 export default function Page() {
-  const { t } = useTranslation()
-  const router = useRouter()
-  
-  const routeToStandardDeductionElection = useAppSelector(state => recommendStandardDeduction(state))
+    const { t } = useTranslation()
```

Comment:
i had to rework how the job list page + its stubs `IncomeFormList` and `IncomeFormListItem` were iterating to support mulitple jobs + mulitple payments